app/controllers/students/attendances_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Students
+  class AttendancesController < ApplicationController
+    before_action :authenticate_user!, :set_user
+
+    def index
+      @attendances = @user.attendances.where(date: from_date..to_date)
+      @attendance_record = @attendances.group(:present).count
+    end
+
+    private
+
+    def set_user
+      @user = User.find(params[:student_id])
```

Comment:
I'd use `set_student` and `@student` if we're talking about a student tho

Reply:
Thanks for the advice. I'll add  `TODO` flag and fix it in the next days. 
Same for the rest of comments like this.